pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -466,6 +464,10 @@ public Boolean get() {
             this.brokerServiceUrl = brokerUrl(config);
             this.brokerServiceUrlTls = brokerUrlTls(config);
 
+            ClusterData clusterData =
+                new ClusterData(webServiceAddress, webServiceAddressTls, brokerServiceUrl, brokerServiceUrlTls);
+            this.webSocketService.setLocalCluster(clusterData);
```

Comment:
The `this.webSocketService` variable may not have been properly initialized?

Reply:
Fixed